public/editor/scripts/main.js

```diff
@@ -47,6 +47,7 @@ require(["jquery", "bowser", "analytics"], function($, bowser, analytics) {
   function onError(err) {
     console.error("[Bramble Error]", err);
     $("#spinner-container").addClass("loading-error");
+    analytics.event({ category : analytics.eventCategories.TROUBLESHOOTING, action : "Editor loading error (black Screen)" });
```

Comment:
I thought about that as well. Is it always a string, or is it some kind of object? I could maybe stringify it and send the first 150 chars or something. I'll see if there are any limits.